daemon/metrics.go

```diff
@@ -26,12 +26,13 @@ func init() {
 		containerActions.WithValues(a).Update(0)
 	}
 	networkActions = ns.NewLabeledTimer("network_actions", "The number of seconds it takes to process each network action", "action")
-	engineVersion = ns.NewLabeledGauge("engine", "The version and commit information for the engine process", metrics.Unit("info"),
+	engineInfo = ns.NewLabeledGauge("engine", "The information related to the engine and the OS it's running on", metrics.Unit("info"),
 		"version",
 		"commit",
 		"architecture",
-		"graph_driver", "kernel",
-		"os",
+		"graph_driver",
```

Comment:
Is this tied to node identity in a swarm? It looks like we have a swarm `NodeID` and an ID in `docker info`. I am not sure about our intention of supporting the id in `docker info`. For the most part, it should be stable but it is tied to a signing system (jwt) that we don't really use.

cc @dmcgowan